faq/prooffacehelp.php

```diff
@@ -613,19 +514,17 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 ?>
 
 <TR><TD><B>Font Size</B>
-<BR>Selection Menu</TD><TD>Select a point size (pt) for the current font
+<BR>Selection Menu</TD><TD class='top-align'>Select a point size (pt) for the current font
 from the dropdown menu to change the font size of the proofreading text.
 </TD></TR>
 
 <TR><TD VALIGN="TOP"><B>Font Face</B>
-<BR>Selection Menu</TD><TD>Select a named font from the dropdown menu.  If the font is
-installed on your system, the proofreading text will change to the selected font.  If the
-font is not installed on your system, your system will either automatically select a
-font in the same family or leave the current font unchanged.
+<BR>Selection Menu</TD><TD class='top-align'>Select a font from the dropdown menu. The choices are
+Browser Default, DP Sans Mono, DejaVu Sans Mono. If you choose a font installed
```

Comment:
I would lean towards not mentioning specific fonts, because then we need to keep them in sync if we add/remove some later. I don't feel super strong about it, however.

Reply:
I have no strong objection to removing the list of fonts. Honestly, hopefully that part of the enhanced interface is reasonably self-explanatory. Done.